doc/BenchmarkingStandard.md

```diff
@@ -22,61 +22,71 @@ The minimum standards implementation is to provide a JSON capabilities file with
 
 ## Capabilities file
 
-The capabilities file is a JSON that the application bundles to describe which standard features it is offering and how it can be used.
+### Description
 
-### Specification
+The capabilities file is a JSON that the application bundles to describe which standard features it is offering and how it can be used.
 
-Platform independent JSON description of this form:
+Here's an example of capabilities JSON file:
 
 ```json
     {
-        "name": "app name",
-        "description": "free form optional description of the app",
+        "name": "application.name",
+        "description": "Optional description of the application",
+        "std_version": "1.0",
```

Comment:
This is mostly meant for scripts and programs, not for human consumption, so I'd want to keep this a number, rather than a string.